packages/web-runtime/themes/owncloud/theme.json

```diff
@@ -122,12 +119,18 @@
           "swatch-primary-hover": "#7bafef",
           "swatch-primary-muted": "",
           "swatch-primary-gradient": "#4e85c8",
-          "swatch-success-default": "#04b253",
-          "swatch-success-hover": "",
-          "swatch-success-muted": "",
-          "swatch-warning-default": "",
-          "swatch-warning-hover": "",
-          "swatch-warning-muted": "",
+          "swatch-success-background": "rgba(0, 188, 140, 0)",
```

Comment:
Please move the `background` swatches into your `OcAlert` PR as well. Not needed in this PR.